src/core/Physics.H

```diff
@@ -16,8 +15,8 @@ public:
 
     //! Initialize the fields for the physics/problem
     virtual void initialize_fields(
-        const amrex::Geometry& geom,
-        LevelData& leveldata) const = 0;
+        int level,
+        const amrex::Geometry& geom) const = 0;
```

Comment:
`amrex::Geometry` can go away too, the classes can access `amrex::AmrMesh`. `level` is needed because we call this from within `MakeNewLevelFromScratch`